src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -27,6 +27,13 @@
     private final Person toAdd;
 
     /**
+     * This method will return true for command types that mutate the data. e.g. AddCommand
+     */
```

Comment:
Coding standard violation:exclamation: The first statement of the header comment does not follow the prescribed style. See point 4, 3rd bullet point of the [[Java Coding Standard: Comments](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#comments)]